tests/test_build_latex.py

```diff
@@ -1383,3 +1383,19 @@ def test_default_latex_documents():
     expected = [('index', 'stasi.tex', 'STASI™ Documentation',
                  r"Wolfgang Schäuble \& G'Beckstein.\@{}", 'manual')]
     assert default_latex_documents(config) == expected
+
+
+@skip_if_requested
+@skip_if_stylefiles_notfound
+@pytest.mark.sphinx('latex', testroot='latex-includegraphics',
+    confoverrides={'latex_documents': [
+                       ('index', 'python.tex', 'Test PDF build',
+                        'Georg Brandl', 'howto')
+                   ]})
+def test_includegraphics_oversized(app, status, warning):
+    app.builder.build_all()
+    print(status.getvalue())
+    print(warning.getvalue())
+    compile_latex_document(app)
+    result = (app.outdir / 'pdflatex/python-dimensions.txt').text(encoding='utf8')
+    print(result)
```

Comment:
I hesitated about analyzing the output file with dimension informations from Python, but finally did it all from internal LaTeX code (see the conf.py)

Reply:
I think this debug print would not display any information even if failed...

Reply:
You are right (of course :) ).